yarn.lock

```diff
@@ -2,6 +2,14 @@
 # yarn lockfile v1
```

Comment:
I don't have much experience with `yarn.lock` files, so I don't know if this is a big diff. All I did was `npx yarn add deprecated-react-native-prop-types`, and this is what got generated. I know lock files can be finicky, so feel free to recreate this file yourself if I did something wrong.